.github/workflows/container-build.yml

```diff
@@ -6,6 +6,7 @@ on:
       digest:
         description: "Container image digest"
         value: ${{jobs.build.outputs.digest}}
+
```

Comment:
(opening here, because GH doesn't allow to comment in non-changed line)

Nit: this workflow  name could be better `name: Build container image and generate SBOMs on every change` for clarity